malmike/springbootbucketlistapi
# #159294803 Project setup

#### What does this PR do?
It involves the initial steps of setting up the project

#### Description of Task to be completed?
- Setup GitHub repository
- Update the readme
- Setup project structure

#### How should this be manually tested?
- Ensure that you have java jdk, maven installed. 
- The project can be [cloned](https://github.com/malmike/springbootbucketlistapi.git).
- Change into the project directory using `cd project_name` and change to the git branch using `git checkout ch-project-setup-159294803`. 
- You can then run `mvn package && java -jar target/springbootbucketlistapi-0.0.1-SNAPSHOT.jar`

#### What are the relevant pivotal tracker stories?
[#159294803](https://www.pivotaltracker.com/story/show/159294803)
